projects/igniteui-angular/src/lib/chips/chip.component.ts

```diff
@@ -927,6 +914,10 @@ export class IgxChipComponent extends DisplayDensityBase implements OnDestroy {
         }
     }
 
+    public ngOnInit(): void {
+        this.computedStyles = this.document.defaultView.getComputedStyle(this.nativeElement);
```

Comment:
I suppose `document` can be null in a SSR environment? Shouldn't there be a more exhaustive check here?

Reply:
I think that it is safe to use the injected document. We do use it within the grid and SSR has no problem